random-beans/pom.xml

```diff
@@ -101,6 +101,12 @@
             <artifactId>junit-dataprovider</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.projectlombok</groupId>
+            <artifactId>lombok</artifactId>
+            <version>1.16.8</version>
+            <scope>provided</scope>
```

Comment:
i think it would not effect the target classed at all, but you´re right. we want 100% nothing less 👍